### DISCUSSION TITLE
CipherOS sync problem

### DISCUSSION THREAD

**Discussion by BhashaRana** [ROM-builders/temporary#18310] [General]
I tried syncing CipherOS here: https://cirrus-ci.com/task/6136839963148288

It fails to sync and complains about revision lineage-18.1 in LineageOS/android_vendor_qcom_opensource_libfmjni not found , whereas it does exist 

So I assume a clean sync is required?
If any solution please give thanks

**Comment by Apon77** [comment]
your local manifests has problem, it has no proper remote name for libjnifm repo, please check and fix it locally first.

**Reply by BhashaRana** [reply]
Cool, thanks I am a noob
